SPECS-EXTENDED/pcp/pcp.spec

```
@@ -2,15 +2,14 @@
 
 Name:    pcp
 Version: 5.1.1
-Release: 3%{?dist}
+Release: 4%{?dist}
 Summary: System-level performance monitoring and performance management
 License: GPLv2+ and LGPLv2+ and CC-BY
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:     https://pcp.io
 
-%global  bintray https://bintray.com/artifact/download
-Source0: %{bintray}/pcp/source/pcp-%{version}.src.tar.gz
+Source: https://github.com/performancecopilot/pcp/archive/refs/tags/5.1.1.tar.gz
```

The file name doesn't match the one from the signatures anymore. I'm guessing this should help:
```suggestion
Source: https://github.com/performancecopilot/pcp/archive/refs/tags/5.1.1.tar.gz#/%{name}-%{version}.tar.gz
```